fe/fe-core/src/main/java/org/apache/doris/catalog/Type.java

```diff
@@ -77,6 +77,7 @@
     public static final ScalarType HLL = ScalarType.createHllType();
     public static final ScalarType CHAR = (ScalarType) ScalarType.createCharType(-1);
     public static final ScalarType BITMAP = new ScalarType(PrimitiveType.BITMAP);
+    public static final ScalarType BLOOMFILTER = new ScalarType(PrimitiveType.BLOOMFILTER);
```

Comment:
No need to add a new type bloomfilter
This bitmap is Type Bitmap, not index bitmap